src/Riok.Mapperly/Descriptors/MappingBodyBuilders/BuilderContext/MembersMappingBuilderContext.cs

```diff
@@ -117,9 +115,25 @@ private HashSet<string> GetSourceMemberNames()
 
     private Dictionary<string, List<PropertyMappingConfiguration>> GetMemberConfigurations()
     {
-        return BuilderContext.Configuration.Properties.ExplicitMappings
-            .GroupBy(x => x.Target.Path.First())
-            .ToDictionary(x => x.Key, x => x.ToList());
+        var simpleMappings = BuilderContext.Configuration.Properties.ExplicitMappings.GroupBy(x => x.Target.Path.First());
+
+        var wildcardMappings = BuilderContext.Configuration.Properties.NestedMappings
+            .SelectMany(x =>
+            {
+                return BuilderContext.SymbolAccessor
+                    .GetAllAccessibleMappableMembers(Mapping.SourceType)
+                    .Where(i => i.Name == x.Source.Path.Last())
+                    .SelectMany(i => BuilderContext.SymbolAccessor.GetAllAccessibleMappableMembers(i.Type))
+                    .Select(i =>
+                    {
+                        var list = x.Source.Path.ToList();
+                        list.Add(i.Name);
+                        return new PropertyMappingConfiguration(new StringMemberPath(list), new StringMemberPath(new[] { i.Name }));
+                    });
```

Comment:
I think mixing these with the manually configured mappings won't fully work...
IMO this needs to be handled separately. If no matching source member could be found, the member should also be looked up in these `MapNestedProperties`.